exercises/nth-prime/src/example.c

```diff
@@ -0,0 +1,32 @@
+#include "nth_prime.h"
+#include <stdbool.h>
+#include <math.h>
+
+static bool is_prime(uint32_t n)
+{
+  for(uint32_t i = (uint32_t)sqrt(n); i > 1; i--) {
```

Comment:
Sieving would be faster for finding a large range of primes, but this is just checking one. The fastest ways to check a single prime are either using the [probabilistic primality testers](https://en.wikipedia.org/wiki/Miller%E2%80%93Rabin_primality_test) with the correct set of witnesses to make the algorithms deterministic, or by trial division (like above). You _can_ speed up trial division by reducing the number of checks through wheel factorization, like I did [here](https://github.com/patricksjackson/primes-cpp/blob/master/primes.cpp#L288), but even compared to the above it is only a speed bump of ~4x.